frontend/src/components/AccountReceiveTable.vue

```diff
@@ -118,7 +118,10 @@
                       </router-link>
                     </base-tooltip>
                   </div>
-                  <div @click="copyAddress(props.row.receiver, 'Receiver')" class="cursor-pointer copy-icon-parent">
+                  <div
```

Comment:
_(arbitrary comment location to start a thread)_

> I think we should include some copy somewhere (maybe below the title, or maybe in an (i)-button mouseover tooltip thing) that explains the send history is only for this device, is stored locally, and can't be found by looking on-chain

Reply:
Added an i tootip

Reply:
Can we put it next to the "Send Funds" table header instead of the page header, i.e. the highlighted text here? 
<img width="834" alt="image" src="https://user-images.githubusercontent.com/17163988/221745417-6b54af30-7195-4bae-8bfe-60f3a590f8fb.png">

Reply:
Yes will move it now